v1-1/learn/events.md

```diff
@@ -67,6 +67,153 @@ redirect_from:
                   
 </table>
 
+<h2>Past Events</h2>
+
+
+<table class="cEventTable cConferencesList" style="width:100%;">      
+                  <tr class="event-expiry" style="display:none" data-expiry="">
+                    <td class="cEventDateContainer"><span class="cEventDate">December 12, 2019</span>
+                      <p class="cEventLocation">Mountain View, California, USA</p>
+                      </td>
+                      <td class="cEventDetail"><a target="_blank" href="https://www.meetup.com/Silicon-Valley-Cloud-Native-and-Kubernetes-Meetup/events/265530711/"><h4>Cloud-Native and Kubernetes Meetup in Silicon Valley</h4></a>
+                          <h5>An Introduction to the Ballerina Programming Language</h5>
+                        <b>Anjana Fernando,</b> Director of Developer Relations - CTO Office, WSO2
+                 </td>
+               <td class="cEventURL"><a class="cEventRegistration" href="https://www.meetup.com/Silicon-Valley-Cloud-Native-and-Kubernetes-Meetup/events/265530711/" target="_blank">More Info</a></td> 
```

Comment:
If we are using the "More Info" text to link to the Conference site, we can remove that column and link the name of the conference itself to link to the conference site.

E.g., "[DeveloperWeek 2020](https://www.developerweek.com/conference/conference-schedule/)"

Reply:
The name of the event links to the main event page and the more info button usually links to the place that shows the abstract of the talk.

Reply:
Then, shall we add the "More info" link also under the 2nd column after mentioning the name of the speaker? I feel it is redundant to have a whole column for that link?

Reply:
If we move the button to the same column I think there'll be a lot of white space on the right-hand side (see how it looks here: https://ballerina.io/events/) That's why the button is positioned there in a different column. If we move it to the same column @madhuramendis will have to redesign the listing page, which IMO is unnecessary. WDYT?

Reply:
+1 to redesign after getting feedback on this from the Team.

Reply:
I don't see a big issue in the current design and both approaches are common in other similar websites too. So IMO, if we really want this re-design, let's do it properly in a separate PR. Since this PR already adheres to the current template, let's go ahead and merge it rather than delaying these details till a design revamp.

Reply:
Created https://github.com/ballerina-platform/ballerina-dev-website/issues/189 for redesigning the Events section.

Therefore, +1 to merge this.